contracts/DappStakingPool.sol

```diff
@@ -223,6 +223,8 @@ contract DappStakingPool is OwnableUpgradeable, ITransferPositionCallback {
         userInfo.lpAmount = userInfo.lpAmount.sub(amount);
         userInfo.rewardDebt = userInfo.amount.mul(pool.accDappPerShare).div(1e12);
         IERC20(dappBntPoolAnchor).transfer(msg.sender, amount);
+
+        if(userInfo.amount == 0) userInfo.positionId = 0;
```

Comment:
Here we don't want this condition, a user can still have a position even if DAPP-BNT is unstaked. Remember, DAPP-BNT is unrelated to positions.